docs/vmalert-tool.md

```diff
@@ -33,6 +33,28 @@ except `promql_expr_test` field. Use `metricsql_expr_test` field name instead. T
 validates and executes [MetricsQL](https://docs.victoriametrics.com/MetricsQL.html) expressions,
 which aren't always backward compatible with [PromQL](https://prometheus.io/docs/prometheus/latest/querying/basics/).
 
+### Limitations
+
+* vmalert-tool evaluates all the groups defined in `rule_files` using `evaluation_interval`[default 1m] instead of `interval` under each rule group.
```

Comment:
@Haleygo could you please elaborate more on this? Why it can't use `interval` field from group?

Reply:
Yeah, because vmalert-tool doesn't actually start rule groups like vmalert, but executes them by hand at each `evaluation_interval`, it's a good way to keep code simple and clean and I don't see it brings any harm.
If user want to test multiple groups with different interval, they can set the `evaluation_interval` with the highest common factor. eg, with `group1: interval 5m` and `group2: interval 1m`, `evaluation_interval` should be set to 1m.
As i understand, `for` should always be equal or bigger than `interval`, otherwise, it doesn't make any difference in vmalert. So in vmalert-tool case, using 1m to evaluate group with `interval: 5m` shouldn't be a problem since it's `for` should be bigger or equal than 5m. 


>eg, with `group1: interval 5m` and `group2: interval 1m`, `evaluation_interval` should be set to 1m.

But I'm not sure to write this to the doc, users might get wrong result if they have rules with `for<interval` now. 

Do you think we should fix this case, support having different group interval in one test?